.github/workflows/build.yml

```diff
@@ -200,8 +189,9 @@ jobs:
           ./ohos-build cargo build --release --target="${{ matrix.target }}"
```

Comment:
While reviewing, I noticed that the OpenHarmony archive should also enable the `streams` feature, since the build-script seems to expect the prebuild archive to have the streams feature enabled. Should I open a separate PR for this, or could you perhaps add this to this PR?

Reply:
Let's add it here and bump mozjs-sys version (we need it anyway to get attestation on release).